### ISSUE TITLE
'Poco/Net/HostEntry.h' file not found

### ISSUE THREAD

**Issue by siteswapjuggler** [bakercp/ofxNetworkUtils#1]
Hello there, your addon seems very nice but I don't succeed using it on the lateste version of OF (v0.10.1). I understand that it's mean to be use with older version, can you confirm this ? perhaps you know of other addon that make the same job or can give me a starting point for me to make it compatible with v0.10.1.

**Comment by bakercp** [comment]
It should still work -- did you include ofxPoco in your addons.make file / project generator?

**Comment by siteswapjuggler** [comment]
You're right I didn't include ofxPoco. It work perfectly now. Perhaps a word about this dependency in the readme would be nice. I got to quick by generating a new project before trying to compile the example where I saw that both addon where actually in it.

Tanks a lot for your work :)
